map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt

```diff
@@ -0,0 +1,43 @@
+package edu.artic.map.rendering
+
+import android.content.Context
+import com.bumptech.glide.Glide
+import com.google.android.gms.maps.model.Tile
+import timber.log.Timber
+import java.io.IOException
+import kotlin.math.pow
+
+/**
+ * Description:
+ */
+class GlideMapTileProvider(private val context: Context,
+                           floor: Int) : BaseMapTileProvider(floor) {
+
+    override fun getAdjustedTile(x: Int, y: Int, zoom: Int): Tile? {
+        return when {
+            x >= 0 && y >= 0 -> {
+                // calculate tile row count
+                val tileXCount = (2.0).pow(zoom)
+
+                // grab which tile based on row, what y number it is in the tile list, x position
+                val tileNumber = (y * tileXCount + x).toInt()
+                Timber.d("Tile X Count $tileXCount with number $tileNumber")
+
+                val path = "http://aic-mobile-tours.artic.edu/sites/default/files/floor-maps/tiles/floor$floor/zoom$zoom/tiles-$tileNumber.png"
```

Comment:
We'll get these urls from the `MapFloorDao` at some future point, iiuc. Please add a todo for switching to `https`.

Reply:
will do so

Reply:
but see above comment

Reply:
fair enough.